.github/workflows/unit-tests.yml

```diff
@@ -5,6 +5,14 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+    paths:
```

Comment:
should the same list apply to integration tests?

Reply:
I hadn't considered that, but that would make sense - @leseb @ashwinb thoughts?

Reply:
Updated in latest patchset